server/graphman/src/resolvers/deployment_mutation.rs

```diff
@@ -81,4 +93,42 @@ impl DeploymentMutation {
         remove::run(&ctx, &name)?;
         Ok(EmptyResponse::new())
     }
+
+    /// Unassign a deployment
+    pub async fn unassign(
+        &self,
+        ctx: &Context<'_>,
+        deployment: DeploymentSelector,
+    ) -> Result<EmptyResponse> {
+        let ctx = GraphmanContext::new(ctx)?;
+        let deployment = deployment.try_into()?;
+
+        unassign::run(&ctx, &deployment)?;
+
+        Ok(EmptyResponse::new())
+    }
+
+    /// Assign or reassign a deployment
+    pub async fn reassign(
+        &self,
+        ctx: &Context<'_>,
+        deployment: DeploymentSelector,
+        node: String,
+    ) -> Result<ReassignResponse> {
+        let ctx = GraphmanContext::new(ctx)?;
+        let deployment = deployment.try_into()?;
+        let node = NodeId::new(node.clone()).map_err(|()| anyhow!("illegal node id `{}`", node))?;
+        reassign::run(&ctx, &deployment, &node)?;
+
+        let mirror = catalog::Mirror::primary_only(ctx.primary_pool);
+        let count = mirror.assignments(&node)?.len();
+        if count == 1 {
+            let warning_msg = format!("warning: this is the only deployment assigned to '{}'. Are you sure it is spelled correctly?",node.as_str());
```

Comment:
We can remove the "warning" prefix from the message because the type only expects warnings. And it does not seem right to ask a question in the message.

```suggestion
            let warning_msg = format!("This is the only deployment assigned to '{}'. Please make sure that the node ID is spelled correctly.",node.as_str());
```